distributed/client.py

```diff
@@ -3156,7 +3156,17 @@ def _graph_to_futures(
             from distributed.protocol.serialize import ToPickle
 
             header, frames = serialize(ToPickle(dsk), on_error="raise")
-            nbytes = len(header) + sum(map(len, frames))
+
+            def safelen(o):
+                try:
+                    return len(o)
+                except TypeError:
+                    # https://github.com/python/cpython/pull/18463/files
+                    if isinstance(o, memoryview) and o.ndim == 0:
+                        return 1
+                    raise
```

Comment:
This should never happen, right?

```suggestion
                    raise  # pragma: nocover
```